community_images/common/latest_tag.py

```diff
@@ -17,6 +17,7 @@ def fetch_tags(image_repo):
         image_repo=image_repo[2:]
 
     resp = requests.get(f"https://registry.hub.docker.com/v1/repositories/{image_repo}/tags")
+    tags = resp;
```

Comment:
why is this required?

Reply:
if resp.status_code <200, the functional would return empty list.

Reply:
can you pls check it once again